### ISSUE TITLE
scripts/system/html/js/list.min.js lacks source

### ISSUE THREAD

**Issue by jonassmedegaard** [overte-org/overte#212]
The file `scripts/system/html/js/list.min.js` is a minified which lacks proper source.

It was apparently introduced in git commit 7ddec78 and since then only renamed a few times.

The code contains references seemingly unrelated to Overte project, including "timoxley-to-array/index.js", "javve-events/index.js" and "javve-get-by-class/index.js".

Is this file used anywhere? Is it understood what it does, fully? What is its copyright and licensing?

**Comment by ksuprynowicz** [comment]
Thanks for noticing this!
It's list.js, released on MIT license:
https://github.com/javve/list.js
I'm not sure if it's still used, but it was intended to be used by entity list in Create App.
What would be the best way to add the license in your opinion? I think MIT and Apache 2.0 are compatible.

**Comment by jonassmedegaard** [comment]
yes, the (Expat variant of) MIT license(s) is compatible with Apache-2.0, but requires that "above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software".

I suggest to both a) include the file `LICENSE` from the list.js project,. renamed as `list.LICENSE`, and also b) include a compact header within the `list.js file itself summarizing copyright and licensing and referencing upstream project, e.g. similar to how it is done in the jquery file.
